projects.html

```diff
@@ -3,52 +3,105 @@
+            <p>Emitii is an online time tracking tool and analysis tool designed for software development
+              teams
+              <a href="http://emitii.com/" class="rnw-link" target="_blank">Learn more</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="column is-3">
+      <div class="card is-shady">
+        <div class="card-image">
+          <img src="/images/talk-ruby.png">
+        </div>
+        <div class="card-content">
+          <div class="content">
+            <h4><a href="http://talk-ruby.com/" target="_blank">Talk Ruby</a></h4>
+            <p>Talk ruby is an online news aggregator about ruby and ruby discussion platform for ruby
+              developers
+              <a href="http://talk-ruby.com/" class="rnw-link" target="_blank">Learn more</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="column is-3">
+      <div class="card is-shady">
+        <div class="card-image">
+          <img src="/images/cloudfactory_logo.png">
+        </div>
+        <div class="card-content">
+          <div class="content">
+            <h4><a href="https://www.cloudfactory.com/" target="_blank">Cloudfactory</a></h4>
+            <p>CloudFactory company is a Distributed Workforce and
+              Enterprise Crowdsourcing Platform
+              <a href="https://www.cloudfactory.com/" class="rnw-link" target="_blank">Learn more</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="column is-3">
+      <div class="card is-shady">
+        <div class="card-image">
+          <img src="/images/ruby_in_tab.png">
+        </div>
+        <div class="card-content">
+          <div class="content">
+            <h4><a href="http://rohitrox.github.io/ruby_in_tab/">Ruby In Tab</a></h4>
+            <p>PhoneGap based android app that enables you to play ruby in your android devices.
+              <a href="http://rohitrox.github.io/ruby_in_tab/" class="rnw-link" target="_blank">Learn more</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="columns features projects">
+    <div class="column is-3">
+      <div class="card is-shady">
+        <div class="card-image">
+          <img src="/images/mero_campus.png">
+        </div>
+        <div class="card-content">
+          <div class="content">
+            <h4><a href="http://www.merocampus.com" target="_blank">Mero Campus </a></h4>
+            <p>an Online Marketplace for College Admissions in Nepal.
+              <a href="http://www.merocampus.com" class="rnw-link" target="_blank">Learn more</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="column is-3">
+      <div class="add-project">
+        <div class="card-content">
+          <div class="content">
+            <h4 class="has-text-centered"><a href="https://github.com/RubyNepal/rubynepal.github.io">
```

Comment:
change link to https://github.com/RubyNepal/rubynepal.github.io/compare

Reply:
okay, I also created projects.yml file and all data related to projects are there from now :)

Reply:
👍